### ISSUE TITLE
Make MiunieUser per guild instead of global

### ISSUE THREAD

**Issue by petrspelos** [discord-bot-tutorial/Miunie#45]
Currently, `MiunieUser` is a "user account" object that is essentially global. This means Miunie will return for a single given user the same user account no matter what guild they're using the command in.

The ideal solution would be to have these accounts guild based.
A significant thing about this is to also store the Guild's ID inside the user account, since some of our services (for example `ls` will became simpler that way).

Basically, let's trade a bit of data storage for an easier data access.
